### PR TITLE
refactored

### PR DESCRIPTION
**Pull request by Belrestro**

### CODE REVIEW

**Comment by artemderkach** [comment]
@Belrestro can you please add description/comment to your code, so other can read and understand it

File: public/js/Canvas.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    this.grid = grid;
+    this.stage = new Stage({
+      container: 'canvas-container',
+      width: width * scale,
+      height: height * scale
+    });
+    // we need to add any canvas object to certain layer, so we need to store links to all of them
+    this.layers = {};
+    this.addLayer('background');
+    this.drawGrid();
```
Comment (artemderkach) [inline public/js/Canvas.js:16]: why add `this.drawGrid()` if then you use `drawGrid()` method

File: public/js/Snake.js [hunk 2]
```diff
@@ -1,34 +1,29 @@
 import _ from 'lodash';
-import Konva from 'konva';
+import {Group, Rect} from 'konva';
 
 export default class Snake {
-  constructor(data, grid) {
-    const {color, id, path} = data;
-    const {scale, layers} = grid;
+  constructor({color, id, path}, grid) {
```
Comment (artemderkach) [inline public/js/Snake.js:5]: what is `path` ?

**Comment by artemderkach** [comment]
@Belrestro use UPPERCASE for `const` variables

**Comment by Belrestro** [comment]
@ArtemDerkach Use your code styling, I'll use mine on front.

**Review by artemderkach — APPROVED** [review]

**Pull request merged**